segmentation/utils/spatial_analysis_utils_test.py

```diff
@@ -0,0 +1,314 @@
+                                                    30: "fov"}, axis=1)
+        return all_patient_data
+    elif typeofencrichment == "positive":
+        all_patient_data_pos = pd.DataFrame(np.zeros((80, 31)))
+        # Assigning values to the patient label and cell label columns
+        all_patient_data_pos[30] = "Point8"
+        all_patient_data_pos[24] = np.arange(len(all_patient_data_pos[1])) + 1
+        # We create 8 cells positive for column index 2, and 8 cells positive for column index 3.
+        # These are within the dist_lim in dist_mat_pos (positive enrichment distance matrix).
+        all_patient_data_pos.iloc[0:8, 2] = 1
+        all_patient_data_pos.iloc[10:18, 3] = 1
+        # We create 4 cells in column index 2 and column index 3 that are also positive
+        # for their respective markers.
+        all_patient_data_pos.iloc[28:32, 2] = 1
+        all_patient_data_pos.iloc[32:36, 3] = 1
+        # Assign column names to columns not for markers (columns to be excluded)
+        all_patient_data_pos = all_patient_data_pos.rename({0: 'cell_size', 1: 'Background', 14: "HH3",
+                                                            23: "summed_channel", 24: "label", 25: "area",
+                                                            26: "eccentricity", 27: "major_axis_length",
+                                                            28: "minor_axis_length", 29: "perimeter",
+                                                            30: "fov"}, axis=1)
+        return all_patient_data_pos
+    elif typeofencrichment == "negative":
+        all_patient_data_neg = pd.DataFrame(np.zeros((60, 31)))
+        # Assigning values to the patient label and cell label columns
+        all_patient_data_neg[30] = "Point8"
+        all_patient_data_neg[24] = np.arange(len(all_patient_data_neg[1])) + 1
+        # We create two groups of 20 cells positive for marker 1 (in column index 2)
+        # and marker 2 (in column index 3) respectively.
+        # The two populations are not within the dist_lim in dist_mat_neg (negative enrichment distance matrix)
+        all_patient_data_neg.iloc[0:20, 2] = 1
+        all_patient_data_neg.iloc[20:40, 3] = 1
+        # Assign column names to columns not for markers (columns to be excluded)
+        all_patient_data_neg = all_patient_data_neg.rename({0: 'cell_size', 1: 'Background', 14: "HH3",
+                                                            23: "summed_channel", 24: "label", 25: "area",
+                                                            26: "eccentricity", 27: "major_axis_length",
+                                                            28: "minor_axis_length", 29: "perimeter",
+                                                            30: "fov"}, axis=1)
+        return all_patient_data_neg
+
+
+def make_threshold_mat():
+    thresh = pd.DataFrame(np.zeros((20, 2)))
+    thresh.iloc[:, 1] = .5
+    return thresh
+
+
+def make_test_closenum():
```

Comment:
This function doesn't make an example closenum, it makes example data for closenum. See if you can rename it so it's more clear what the point of this function is